README.md

```diff
@@ -2,71 +2,171 @@
 
 CLI for reporting events to Faros platform.
 
-The script provides all the necessary instrumentation for CI/CD pipelines by sending events to Faros platform
+- [Installation](#installation)
+- [Quickstart](#quickstart)
+- [Instrumenting CI/CD Pipelines](#instrumenting-cicd-pipelines)
+  - [Reporting Builds with Commits (Basic)](#reporting-builds-with-commits-basic)
+  - [Reporting Deployments with Commits (Basic)](#reporting-deployments-with-commits-basic)
+  - [Reporting Builds & Deployments in Parts (Advanced)](#reporting-builds--deployments-in-parts-advanced)
+  - [Reporting Builds & Deployments with Commits & Artifacts (Advanced)](#reporting-builds--deployments-with-commits--artifacts-advanced)
+- [Code Quality](#code-quality)
+  - [Reporting test execution results](#reporting-test-execution-results)
+- [Usage with Faros Community Edition](#usage-with-faros-community-edition)
+- [Arguments](#arguments)
+- [Tips](#tips)
```

Comment:
1. Expand `Tips` section here
2. Move `Usage with Faros Community Edition` to `Tips` section
3. `Usage with Faros Community Edition` - remove the example and just mention the necessary flag